src/services/inventoryService.ts

```diff
@@ -136,20 +153,40 @@ export const addItem = async (
                         }
                     };
                 case "Items": {
-                    const inventory = await getInventory(accountId);
-                    const miscItemChanges = [
-                        {
-                            ItemType: typeName,
-                            ItemCount: quantity
-                        } satisfies IMiscItem
-                    ];
-                    addMiscItems(inventory, miscItemChanges);
-                    await inventory.save();
-                    return {
-                        InventoryChanges: {
-                            MiscItems: miscItemChanges
+                    switch (typeName.substr(1).split("/")[3]) {
+                        case "ShipDecos": {
+                            const inventory = await getInventory(accountId);
+                            const changes = [
+                                {
+                                    ItemType: typeName,
+                                    ItemCount: quantity
+                                } satisfies IMiscItem
+                            ];
+                            addShipDecorations(inventory, changes);
+                            await inventory.save();
+                            return {
+                                InventoryChanges: {
+                                    ShipDecorations: changes
+                                }
+                            };
                         }
-                    };
+                        default: {
```

Comment:
why do we save default items as miscItems? is everything that is a default item a misc item?

Reply:
I don't know, but I just wanted to keep the same behaviour for everything that is not a ship decoration.